packages/@uppy/xhr-upload/src/index.ts

```diff
@@ -440,18 +440,25 @@ export default class XHRUpload<
       opts.allowedMetaFields,
       file.meta,
     )
+    // When you .use(XHR) with .use(Transloadit, { onlyRemoteFiles: true }),
+    // local files are uploaded with this plugin and remote files with the Transloadit plugin.
+    // Since the Transloadit plugin uses the tus plugin underneath, it's possible to have file.tus
+    // even though we are in this plugin.
+    // @ts-expect-error typed in @uppy/tus
```

Comment:
is this expect error needed?
```suggestion
```

Reply:
we are in `xhr-upload`, `file.tus` only exists if you install `@uppy/tus`: 

https://github.com/transloadit/uppy/blob/8f53f6a42ba75a1601ffca02190a1ddc38abc958/packages/%40uppy/tus/src/index.ts#L105-L110